src/docs/product/data-management-settings/dynamic-sampling/dsla/getting-started.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Getting Started
+sidebar_order: 10
+noindex: true
+description: "Learn how to get started with dynamic sampling."
+---
+
+<Include name="limited-avail-note.mdx" />
+
+<Note>
+
+To set up or configure our **Dynamic Sampling** feature, you need to be an Owner, Manager, or Admin of your Sentry organization.
+
+</Note>
+
+## Current Limitations
+
+The **Dynamic Sampling** feature is currently only available for the following SDKs:
+
+<Include name="sampling-supported-sdks.mdx" />
+
+If your application relies on any Sentry SDK that isn't specified above, then you won’t be able to use Sentry's **Dynamic Sampling**.
+
+There are other limitations that you should be aware of when setting up dynamic sampling:
+
+- [Trace rules can only select based on the attributes of the initial transaction](/product/data-management-settings/dynamic-sampling/dsla/current-limitations/#trace-rules-can-only-select-based-on-the-attributes-of-the-initial-transaction)
+- [Changing trace attributes in secondary transactions](/product/data-management-settings/dynamic-sampling/dsla/current-limitations/#changing-trace-attributes-in-secondary-transactions)
+- [Limited number of rule conditions](/product/data-management-settings/dynamic-sampling/dsla/current-limitations/#limited-number-of-rule-conditions)
+- [Arbitrary logical composition for rule conditions](/product/data-management-settings/dynamic-sampling/dsla/current-limitations/#arbitrary-logical-composition-for-rule-conditions)
+
+For more detailed information about each of these, check out our [full Current Limitations documentation](/product/data-management-settings/dynamic-sampling/dsla/current-limitations/).
+
+## 1. Update Sentry SDK Versions
```

Comment:
afaik Dynamic Sampling will still be available for unsupported platforms but may not provide good results as for supported platforms. Should we mention this?

Reply:
@ale-cota your call

Reply:
Per @ale-cota, we're going to leave this as the platforms that are 100% supported.